src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -148,6 +148,13 @@
           UseHardlinksIfPossible="true" />
   </Target>
 
-  <Import Project="$(RepositoryEngineeringDir)sdkArchiveDiff.targets" Condition="'$(PortableBuild)' == 'true' and '$(PgoInstrument)' != 'true' and '$(DotNetBuildSourceOnly)' != 'true'" />
+  <Target Name="RunUnifiedBuildValidation"
+          AfterTargets="Build"
+          Condition="'$(PortableBuild)' == 'true' and '$(PgoInstrument)' != 'true' and '$(DotNetBuildSourceOnly)' != 'true'" >
+    <MSBuild Projects="../test/Microsoft.DotNet.UnifiedBuild.Tests/Microsoft.DotNet.UnifiedBuild.Tests.csproj"
+             Targets="VSTest"
+             ContinueOnError="true"
+             Properties="VsTestUseMSBuildOutput=true" />
```

Comment:
Unsure if you saw my previous comment about this. Can we set `VsTestUseMSBuildOutput` directly in the test project and remove it from here?

Reply:
Yes, I missed that one, sorry.